index.js

```diff
@@ -69,6 +69,12 @@ async function run() {
       await configureSteampipePlugins(plugins);
     }
 
+   if (Object.keys(plugins).length > 0) {
+      core.debug(`Listing plugins to check their status and initialize information_schema`);
```

Comment:
```suggestion
      core.debug(`Listing plugins to check their status`);
```

Given that the `information_schema` not being initialized is a bug in the CLI (which should get fixed in https://github.com/turbot/steampipe/issues/4456, I think we should anticipate that fix coming in a new version of the CLI and this message can focus on checking plugin status.

Reply:
Also, could we combine the lines in this `if` statement with the one above, since they check for the same thing?

Reply:
> this message can focus on checking plugin status.

Agreed. I committed your suggestion.

I'll address your second comment in the main conversation because this thread is marked as resolved.